openidcpy/__init__.py

```diff
@@ -1 +1 @@
-from client import OidcClient
\ No newline at end of file
+from .client import OidcClient
```

Comment:
Relative paths are not a great way to resolve dependencies imo. Use `openidcpy.client` instead